framework/framework.go

```diff
@@ -187,6 +187,61 @@ func (f *Framework) UpdateFunc(oldObj, newObj interface{}) {
 	}
 }
 
+// ProcessEvents takes the event channels created by the operatorkit informer
+// and executes the framework's event functions accordingly.
+func (f *Framework) ProcessEvents(ctx context.Context, deleteChan chan watch.Event, updateChan chan watch.Event, errChan chan error) {
```

Comment:
This should then actually be above `Framework.UpdateFunc`.

Reply:
Bam!